src/pages/profile.js

```diff
@@ -0,0 +1,149 @@
+import Avatar from "../assets/avatar-profile.png";
+import NavBar from "../components/navBar";
+import Pencil from "../assets/pencil-icon.png";
+import Suitcase from "../assets/role_icon.png";
+import Pin from "../assets/location_icon.png";
+import Discord from "../assets/discord_icon.png";
+import Envelope from "../assets/email_icon.png";
+import InvisibleLab from "../assets/invisible_lab_icon.png";
+import Onda from "../assets/onda_logo.png";
+import Sub from "../assets/sub_logo.png";
+import PinkRoom from "../assets/pink_room_logo.png";
+import Universe from "../assets/universe.jpg";
+
+import List from "../components/list";
+import Info from "../components/info";
+import {
+  getAnswers,
+  getSkills,
+  getHobbies,
+  getUserData,
+  getQuestion,
+} from "../utils/getters";
+import { useState, useEffect } from "react";
+
+const current_user_id = localStorage.getItem("current");
+
+const companyLogos = {
+  Subvisual: Sub,
+  "Invisible lab": InvisibleLab,
+  Onda: Onda,
+  "Pink Room": PinkRoom,
+  Universe: Universe,
+};
+
+const formatDate = (dateString) => {
+  const date = new Date(dateString);
+  const options = { month: "short", day: "2-digit" };
+  return date.toLocaleString("en-US", options);
+};
+
+function Profile() {
+  const [answers, setAnswers] = useState([]);
+  const [userData, setUserData] = useState([]);
+  const [skills, setSkills] = useState([]);
+  const [hobbies, setHobbies] = useState([]);
+  const [company, setCompany] = useState([]);
```

Comment:
Is this supposed to be an array by default?

Reply:
yes because I have to map those